packages/vanilla/src/complex/array/index.ts

```diff
@@ -28,9 +28,8 @@ import {
   rankWith
 } from '@jsonforms/core';
 import ArrayControlRenderer from './ArrayControlRenderer';
-import { ArrayControl } from './ArrayControl';
 
-export { ArrayControl, ArrayControlRenderer };
+export {  ArrayControlRenderer };
```

Comment:
Similarily to the comment in `ArrayControlRenderer.tsx`, I think we should export `ArrayControl` here.
You can get import it together with the `ArrayControlRenderer` like this:
`import ArrayControlRenderer, { ArrayControl } from './ArrayControlRenderer';`
and the change the export to
`export {  ArrayControlRenderer, ArrayControl };`